.github/workflows/integration-tests.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       ATALA_GITHUB_ACTOR: ${{ secrets.ATALA_GITHUB_ACTOR }}
       ATALA_GITHUB_TOKEN: ${{ secrets.ATALA_GITHUB_TOKEN }}
       REPORTS_DIR: "didcomm-v2-mediator-test-suite/target/site/serenity"
+      DIDCOMM_V2_TESTSUITE_VERSION: "v0.1.0"
+      MEDIATOR_DID: "did:peer:2.Ez6LSghwSE437wnDE1pt3X6hVDUQzSjsHzinpX3XFvMjRAm7y.Vz6Mkhh1e5CEYYq6JBUcTZ6Cp2ranCWRrv7Yax3Le4N59R6dd.SeyJ0IjoiZG0iLCJzIjoiaHR0cDovL2xvY2FsaG9zdCIsInIiOltdLCJhIjpbImRpZGNvbW0vdjIiXX0"
```

Comment:
The `"serviceEndpoint" : "http://localhost"` has no port.

Maybe we should use some useful default.
You are replacing this value somewhere on runtime right?